participants/MemoryGenius.md

```diff
@@ -0,0 +1,73 @@
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+*Please tell us your project's name and the Filecoin Slack handles for your teammates.*
+Project Name：MemoryGenius
+Team Slack Handles：https://filecoinproject.slack.com/team/UMMPE56MP
+
+## (1) Brief Project Description
+*Add your project's description here.*
+Taking advantage of IPFS technology to release news about IPFS then acting as IPFS's spokesman to attract more people to understand and approve of IPFS
+
+## (2) Link to Project UI
+*Please include a link to your project application or UI. It does not need to be super polished or completely functional. We will be reviewing this link periodically to verify that your project is eligible for final rewards.*
+www.ipfsnews.net
+
+## (3) What does your application/UI do?
+*Give us a brief description of what your application/UI does and how users are intended to interact with it.*
+Releasing IPFS tendency, user comments/vote feedback
+## (4) If your project is using a curated dataset, which one are you using?
+*You can reference the [list of curated datasets in this repo](https://github.com/filecoin-project/slingshot/blob/master/datasets.md). If you'd like to use a dataset that is not on that list, please nominate your chosen dataset by updating the curated datasets table in this PR. If nominating a dataset, do you own or have exclusive rights to the data you plan to use?*
+Not curated dataset
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+*What sorts of data will you collect? Do you own or have exclusive rights to the data you plan to use? Is the data license available online? Does the data contain any sensitive or illegal information?*
+sql text/video/Blockchain map
+yes
+yes
+no
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+*Give us as realistic an estimate as you possibly can at this stage. We understand it's impossible to have perfect information before really starting your project!*
+400TiB-600TiB
+
+## (7) How are you structuring the data?
+*Is this structured or unstructured data? What data formats, schemas, etc. does this dataset follow?*
+Video and structured data
+
+## (8) What pre-processing are you doing before ingesting the data?
+*How will you prepare this data for ingestion into Filecoin? What size storage deals will you be making? If this is a tabular dataset or directory structure, how will you maintain indices into the data so you can retrieve specific data as needed for your application? This is currently one of the most important steps to being successful with data storage on Filecoin. Feel free to ask for help and/or look at docs for recommendations!*
+20% stock data+80% data to be developed

```

Comment:
Can you elaborate on what data will be developed?   This may need to be reviewed as this becomes clearer.